### DISCUSSION TITLE
Contributor? Introduce Yourself!

### DISCUSSION THREAD

**Discussion by kaustubhavarma** [tattle-made/DAU#69] [General]
**Hello! Welcome to our community!** 🙌

Get started by:

1️⃣ introducing yourself 👋
2️⃣ the work you do 📝
3️⃣ what you'd like to work on here 💻

We'll help you identify relevant issues you can work on if you haven't decided yet  🤝

**Comment by bbansal7** [comment]
Hi ,
I am Bhavyaa from India . I am software developer , mainly hands-on in C++ and C#. I am interested in making internet a safe space. I look forward to contribute to the project in meaningful ways.

**Comment by kaustubhavarma** [comment]
Hi Bhavyaa!

Happy to have you here! We are having an all-hands meeting today for this project at 11AM, you can join here if you're interested: https://meet.jit.si/moderated/9eb69848382fa778a1097fb3f861aa320682b19d13263c592ecee4d3dcb2647b
